src/pages/categories-page/CategoryPage.js

```diff
@@ -0,0 +1,53 @@
+import { React } from "react";
```

Comment:
React is default export and not a named export from react. Please update it.

Reply:
done